Dockerfile-latest

```diff
@@ -7,6 +7,8 @@ RUN yum -y install epel-release centos-release-openshift-origin \
 
 WORKDIR /mnt
 
+COPY apb-wrapper /usr/local/bin/apb-wrapper
```

Comment:
This comes for the apb-base-scripts package installed in the yum command above on the latest image.